PLAYER/StartPlayerTeleport.md

```diff
@@ -16,6 +16,7 @@ void START_PLAYER_TELEPORT(Player player, float x, float y, float z, float headi
 * **z**: 
 * **heading**: 
 * **p5**: 
-* **p6**: 
+* **findCollisionLand**: 
 * **p7**: 
 
+The boolean findCollisionLand teleports the player to land and will not consider the Z coordinate parameter provided by you, it will automatically put the Z coordinate so that you don't fall from sky.
```

Comment:
Argument documentation does not go in a paragraph in the Parameters section - these will be ignored.